test/test-pipe-set-non-blocking.c

```diff
@@ -102,8 +102,7 @@ TEST_IMPL(pipe_set_non_blocking) {
     ASSERT(n == UV_EAGAIN); /* E_NOTIMPL */
     ASSERT(0 == uv_write(&write_req, (uv_stream_t*) &pipe_handle, &buf, 1, write_cb));
     ASSERT_NOT_NULL(write_req.handle);
-    ASSERT(1 == uv_run(uv_default_loop(), UV_RUN_ONCE)); /* queue write_cb */
-    ASSERT(0 == uv_run(uv_default_loop(), UV_RUN_ONCE)); /* process write_cb */
+    ASSERT(0 == uv_run(uv_default_loop(), UV_RUN_ONCE));
```

Comment:
```suggestion
    ASSERT_EQ(0, uv_run(uv_default_loop(), UV_RUN_ONCE));
```

Reply:
I am not changing the code here (just removing a comment), so I was inclined to keep it the same. Particularly since we want to show this change has minimal impact.